api/src/main/java/com/codeforcommunity/rest/subrouter/authenticated/ProtectedUserRouter.java

```diff
@@ -71,6 +71,18 @@ private void registerGetAllUsers(Router router) {
     getAllUsersRoute.handler(this::handleGetAllUsers);
   }
 
+  private void registerDisableAccount(Router router) {
+    Route disableUserAccountRoute = router.post("/disable/:user_id");
```

Comment:
⛏️ 🤔 Is this URI in a RESTful format? Maybe we should do `"user/:user_id/disable"` or `"user/disable/:user_id"`?

Reply:
I'm pretty sure the current route is /user/disable/user_id which aligns with your second option. Would you prefer the first option? @rymaju